src/Containers/ClientShowCase.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from '../Inputs/Button';
+
+const IMAGE_HEIGHT = 30;
+
+export interface ClientProps extends React.HTMLAttributes<HTMLDivElement> {
+    imgData: string[];
+    handleClick: (event: React.MouseEvent<Element, MouseEvent>) => void;
+    buttonText: string;
+    animationTime: number;
+}
+
+export interface ImageListProps {
+    animationTime: number;
+}
+
+export const ClientShowCase: React.FC<ClientProps> = ({
+    imgData,
+    handleClick,
+    buttonText = 'SEE OUR CUSTOMERS',
+    animationTime = 1.5,
+    ...props
+}): React.ReactElement => {
+    return (
+        <ImageListDiv onClick={handleClick} {...props}>
+            <ImageList animationTime={animationTime}>
+                {imgData.map((imgURL: string) => (
+                    <IconListItem key={imgURL}>
+                        <IconImg src={imgURL} />
+                    </IconListItem>
+                ))}
+            </ImageList>
+            <ButtonDiv>
+                <MyButton primary>{buttonText}</MyButton>
+            </ButtonDiv>
+        </ImageListDiv>
+    );
+};
+
+const ImageListDiv = styled.div`
+    width: 100%;
+    height: auto;
+`;
+
+const ImageList = styled.ul<ImageListProps>`
+    display: inline-block;
+    list-style-type: none;
+    text-align: center;
+    ${ImageListDiv}:hover & {
+        transition: filter ${({ animationTime }): number => animationTime}s;
+        filter: blur(7px);
+    }
```

Comment:
Use mixin.transition
I dont think you need these, since a div is a block, which takes up with by default